config/initializers/active_record_log_subscriber.rb

```diff
@@ -0,0 +1,16 @@
+module LogQuerySource
+  def debug(*args, &block)
+    return unless super
+
+    backtrace = Rails.backtrace_cleaner.clean caller
+
+    relevant_caller_line = backtrace.detect do |caller_line|
+      !caller_line.include?('/initializers/')
+    end
+
+    if relevant_caller_line
+      logger.debug("  ↳ #{ relevant_caller_line.sub("#{ Rails.root }/", '') }")
+    end
+  end
+end
+ActiveRecord::LogSubscriber.send :prepend, LogQuerySource
```

Comment:
It is in the parent pull request and therefore even though this pull request doesn't change the file this file is still shown.